dashboards-observability/.cypress/integration/app_analytics.spec.js

```diff
@@ -22,10 +22,14 @@ import {
   trace_one,
   trace_two,
   trace_three,
-  spanQueryPartOne,
-  spanQueryPartTwo,
-  spanQueryPartThree,
-  visName,
+  spanQueryOnePartOne,
+  spanQueryOnePartTwo,
+  spanQueryOnePartThree,
+  spanQueryTwoPartOne,
+  spanQueryTwoPartTwo,
+  spanQueryTwoPartThree,
```

Comment:
np but might be better to loop over array, also why does this need to be split into 3 parts?

Reply:
i see, this is ok but you can also just pass the whole string and use a separate function to split by fixed length and feed to cypress.type()

Reply:
Yea the only problem is cypress would repeatedly miss certain characters so I had to split the string at those places not at a fixed length